lib/form/validation.js

```diff
@@ -130,7 +149,7 @@ var validate = function (e) {
   });
 
   // Run the validation
-  validation = allValidation[inp.parent.type][inp.validation.function](input, settings);
+  validation = required(input, settings, allValidation[inp.parent.type][inp.validation.function]);
```

Comment:
Maybe my problem with this is that you're calling these functions `required`. That's _not_ what they're checking. It's a validation function that checks for required and then checks input validation. Maybe you just need to update the name to something that more accurately reflects what this is doing